fighters/demon/src/acmd/tilts.rs

```diff
@@ -48,6 +49,7 @@ unsafe fn game_attackhi3(fighter: &mut L2CAgentBase) {
         }
     }
     frame(lua_state, 4.0);
+    FT_MOTION_RATE_RANGE(fighter, 4.0, 8.0, 3.0);
```

Comment:
there's no line to reset the motion rate on this, until frame 21 (seems like it should be frame 8) -- this actually shaves off 4.25 frames from the move (25% of the frames between 4 and 21)